packages/block-editor/src/store/selectors.js

```diff
@@ -2647,19 +2647,43 @@ export const __experimentalGetActiveBlockIdByBlockNames = createSelector(
 export function wasBlockJustInserted( state, clientId, source ) {
 	const { lastBlockInserted } = state;
 	return (
-		lastBlockInserted.clientId === clientId &&
+		lastBlockInserted.clientIds?.includes( clientId ) &&
 		lastBlockInserted.source === source
 	);
 }
 
 /**
  * Gets the client id of the last inserted block.
  *
+ * @deprecated
  * @param {Object} state Global application state.
  * @return {string|undefined} Client Id of the last inserted block.
  */
 export function getLastInsertedBlockClientId( state ) {
-	return state?.lastBlockInserted?.clientId;
+	deprecated(
+		'wp.data.select( "core/block-editor" ).getLastInsertedBlockClientId',
+		{
+			since: '15.0',
```

Comment:
Can I get a confidence check that this is the correct release?

Reply:
14.9 was the last release so I believe so.